Create 2025-06-12.md (#251)
department-of-veterans-affairs/developer-portal-content: content/benefits/education-benefits/release-notes/2025-05-13.md

> Changes were made to the API's error processing to adjust for changes implemented by the VA source system.

Reviewer: Could would be a little bit more specific here?

Reviewer: What were the changes? Give the consumer a little bit more detail.

> Changes were made to return a single error message - Veteran not found - when the VA source system returns multiple messages for an incomplete Veteran record. 

Reviewer: ```suggestion
We updated the API's error handling to return a single message — Veteran not found — when the VA source system provides multiple messages for an incomplete record.
```
